doc/man7/RSA-PSS.pod

```diff
@@ -30,15 +30,19 @@ By default no parameter restrictions are placed on the generated key.
 
 =head1 NOTES
 
-The public key format is documented in RFC4055.
+The public key format is documented in RFC 8017, which differs from RFC 4055
+L<section 3.1|https://www.rfc-editor.org/rfc/rfc4055.html#section-3.1>
```

Comment:
That's not right: both RFCs require the explicit ASN.1 NULL to be there. It is the documentation that was incorrect in claiming that the old format corresponds to RFC 4055.